src/main/java/edu/ie3/datamodel/io/processor/EntityProcessor.java

```diff
@@ -337,68 +350,11 @@ protected String processVoltageLevel(VoltageLevel voltageLevel, String fieldName
    *     or empty if an error occurred during processing
    */
   protected Optional<String> handleQuantity(Quantity<?> quantity, String fieldName) {
-
-    Optional<String> normalizedQuantityValue = Optional.empty();
-
-    switch (fieldName) {
-      case "p":
-      case "q":
-      case "energy":
-      case "vTarget":
-      case "vrated":
-      case "sRated":
-      case "eConsAnnual":
-        normalizedQuantityValue = handleProcessorSpecificQuantity(quantity, fieldName);
-        break;
-      case "soc":
-      case "vAng":
-      case "vMag":
-      case "iAAng":
-      case "iBAng":
-      case "iCAng":
-      case "etaConv":
-      case "azimuth":
-      case "height":
-        normalizedQuantityValue = quantityValToOptionalString(quantity);
-        break;
-      case "iAMag":
-      case "iBMag":
-      case "iCMag":
-        normalizedQuantityValue =
-            quantityValToOptionalString(
-                quantity
-                    .asType(ElectricCurrent.class)
-                    .to(StandardUnits.ELECTRIC_CURRENT_MAGNITUDE));
-        break;
-      case "qDot":
-        normalizedQuantityValue =
-            quantityValToOptionalString(
-                quantity.asType(Power.class).to(StandardUnits.Q_DOT_RESULT));
-        break;
-      case "fillLevel":
-        normalizedQuantityValue =
-            quantityValToOptionalString(
-                quantity.asType(Dimensionless.class).to(StandardUnits.FILL_LEVEL));
-        break;
-      case "length":
-        normalizedQuantityValue =
-            quantityValToOptionalString(
-                quantity.asType(Length.class).to(StandardUnits.LINE_LENGTH));
-        break;
-
-      case "feedInTariff":
-        normalizedQuantityValue =
-            quantityValToOptionalString(
-                quantity.asType(EnergyPrice.class).to(StandardUnits.ENERGY_PRICE));
-        break;
-      default:
-        log.error(
-            "Cannot process quantity with value '{}' for field with name {} in model processing!",
-            quantity,
-            fieldName);
-        break;
+    if (specificQuantityFieldNames.contains(fieldName)) {
```

Comment:
While I like this approach it neglects that the provided data might not be in `StandardUnits`. From my point of view you can't always ensure that the provided data is in `StandardUnits` because it's NOT mandatory that they have been read by our processors and transformer to standard units accordingly. Hence I would suggest keeping the distinction between quantities that needs to be normalized and the ones that can just be handled via `quantityValtoOptionalString()`

Reply:
Okay, but then we would have to "standardize" all quantities, we get. I decided not to do it, because the conversion is made within all constructors of our models. So unless you do any reflective magic, all our models will hold the correct unit.